ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/ldesfragment/services/FragmentationExecutorImpl.java

```diff
@@ -72,7 +73,14 @@ private LdesFragment retrieveRootFragmentOfView(ViewName viewName, Observation p
 
 	@EventListener
 	public void handleViewAddedEvent(ViewAddedEvent event) {
+		// TODO create root fragment and refragment all earlier members of collection
```

Comment:
Really not a big fan of adding an additional 2 TODOs in our code. This is why i would prefer to go to the v2 branch already